xbmc/cores/AudioEngine/Sinks/AESinkDirectSound.cpp

```diff
@@ -184,17 +184,15 @@ bool CAESinkDirectSound::Initialize(AEAudioFormat &format, std::string &device)
     return false;
   }
 
-  WAVEFORMATEXTENSIBLE wfxex = {0};
-
   // clamp samplerate between 44100 and 192000
   if (format.m_sampleRate < 44100)
     format.m_sampleRate = 44100;
 
   if (format.m_sampleRate > 192000)
     format.m_sampleRate = 192000;
 
-  //fill waveformatex
-  ZeroMemory(&wfxex, sizeof(WAVEFORMATEXTENSIBLE));
+  // fill waveformatex
+  WAVEFORMATEXTENSIBLE wfxex = {};
```

Comment:
I think this is syntactic sugar for `WAVEFORMATEXTENSIBLE wfxex{};` so I don't think the `=` is necessary

Reply:
I think it is totally equivalent but in the code it seems that the form `= {}` is used more. As you say, I don't think there is any functional difference by using one way or another.

Reply:
I prefer without `=` when possible but it's your work so totally your call. I'll approve either way.